src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/HotReload/ProjectHotReloadSessionManager.cs

```diff
@@ -449,24 +477,25 @@ async Task ApplyHotReloadUpdateInternalAsync()
             }
         }
 
-        private class HotReloadState : IProjectHotReloadSessionCallback
+        internal class HotReloadState : IProjectHotReloadSessionCallback
         {
-            private readonly ProjectHotReloadSessionManager _sessionManager;
-
             public Process? Process { get; set; }
             public IProjectHotReloadSession? Session { get; set; }
 
-            // TODO: Support restarting the session.
-            public bool SupportsRestart => false;
+            public ProjectHotReloadSessionManager SessionManager { get; }
```

Comment:
I can't see why this was made into a public property. Is it actually used? If not, perhaps restore the private readonly field.

Reply:
Restored